ReactWindows/ReactNative.Net46/Views/TextInput/ReactPasswordBoxManager.cs

```diff
@@ -189,7 +191,22 @@ public void SetFontStyle(PasswordBox view, string fontStyleString)
         [ReactProp("placeholder")]
         public void SetPlaceholder(PasswordBox view, string placeholder)
         {
-            throw new NotSupportedException("Placeholder is not supported in WPF.");
+            PlaceholderAdorner.SetText(view, placeholder);
```

Comment:
The adorner is applied by use of attached properties... Here we set the text and indirectly create/attach the adorner by calling the static property accessor and passing the placeholder text...